src/abstractdataframe/reshape.jl

```diff
@@ -194,6 +194,7 @@ function unstack(df::AbstractDataFrame, rowkey::Int, colkey::Int, value::Int)
     # `colkey` integer indicating which column to place along column headers
     # `value` integer indicating which column has values
     refkeycol = CategoricalArray{Union{eltype(df[rowkey]), Missing}}(df[rowkey])
+    droplevels!(refkeycol)
```

Comment:
Better add a comment explaining that we do this for consistency with the case when the column is not specified.

Reply:
fixed